executor/cook/io_helper.py

```diff
@@ -12,13 +12,15 @@
 __stderr_lock__ = Lock()
 
 
-def print_out(string_data, flush=False, newline=True):
+def print_out(data, bytes=False, flush=False, newline=True):
     """Wrapper function that prints to stdout in a thread-safe manner using the __stdout_lock__ lock.
 
     Parameters
     ----------
-    string_data: string
-        The string to output
+    data: string or bytes
+        The data to output
+    bytes: boolean
```

Comment:
When you would pass `True` for `bytes`, is `data` a `bytes` object? If so, can we remove the parameter and just check the input type? This seems pretty error-prone.

Reply:
Alternatively, is there any downside to *always* writing to `stdout.buffer`?

Reply:
Done:
- directly checking the input type for string or not (bytes)
- always always writing to stdout|stderr.buffer